### PR TITLE
Cluster autoscaler feature

### PR DESCRIPTION
**Pull request by rpudlowski93**
PR with AKS autoscaling setup using Terraform scripts. Connected with task #1441
Tests of the autoscaling are done and described in main task (#1441)

### CODE REVIEW

File: feature/aks-cluster-creation/README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ The Azure Kubernetes Service (AKS) is the service for deploying, managing, and s
 - Different network drivers (basic/advance)
 - Different resource groups
 - Possibility to create new network or use existing network and resource group. AKS will just join to the existing network.
+- Cluster autoscaling
```
Comment (mkyc) [inline feature/aks-cluster-creation/README.md:16]: what do you think about moving all terraform file to subdirectory of `design-doc/aks`? That would feel natural for me to later merge it to develop and have all those PoC scripts in there.

Comment (rpudlowski93) [reply feature/aks-cluster-creation/README.md:16]: Nice idea, it's good to have "somewhere" the terraform scripts, even after merged the branch into develop.

Comment (mkyc) [reply feature/aks-cluster-creation/README.md:16]: so would you delete this directory?

File: docs/design-docs/aks/aks.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Azure Kubernetes Service (AKS) is the service for deploying, managing, and scali
 - Different network drivers (basic/advance)
 - Different resource groups
 - Possibility to create new network or use existing network and resource group. AKS will just join to the existing network.
+- Cluster autoscaling
```
Comment (mkyc) [inline docs/design-docs/aks/aks.md:17]: rename this file to README.md

Comment (rpudlowski93) [reply docs/design-docs/aks/aks.md:17]: Done

**Comment by mum0l** [comment]
Terraform apply command ends up with kubeconfig displayed as a last output. I suppose it might be good to put into into the file, and as output, simply show `export KUBECONFIG=$kubeconf` file, so user can simply copy/paste and apply the command. This is also not documented, how to connect into cluster, so will be good to update README.

Beside that everything works properly :)

**Pull request merged**